manifests/plugin/ansible.pp

```diff
@@ -38,6 +42,26 @@
     template_path => 'foreman_proxy/plugin/ansible.yml.erb',
   }
 
+  file { "${::foreman_proxy::var_dir}/ansible":
```

Comment:
The ```var_dir``` isn't created by packaging. It may be best to ensure that in ```foreman_proxy::config``` so all plugins can rely on it.